subworkflows/local/profiling.nf

```diff
@@ -145,7 +145,15 @@ workflow PROFILING {
     if ( params.run_kraken2 && params.run_bracken ) {
         // Remove files from 'pure' kraken2 runs, so only those aligned against Bracken & kraken2 database are used.
         def ch_kraken2_output = KRAKEN2_KRAKEN2.out.report
-            .filter { meta, report -> meta['tool'] == 'bracken' }
+            .filter {
+                meta, report ->
+                    meta['tool'] == 'bracken' && meta['instrument_platform'] != 'OXFORD_NANOPORE'
+                    if ( meta['instrument_platform'] == 'OXFORD_NANOPORE' ) log.warn "[nf-core/taxprofiler] Bracken has not been evaluated for Nanopore data. Skipping Bracken for sample ${meta.id}."
+
+
+            }
```

Comment:
I had missed that you applied this filter to the data that will serve as the input for Bracken only. So in that sense the tool check is actually not needed. I do think the value to be returned from a closure needs to be the last statement, though. Have you tested this?